dataservice/extensions/flask_indexd.py

```diff
@@ -224,20 +257,25 @@ def delete(self, record):
             return record
 
         if record.rev is None:
-            r = self.session.get(self.url + record.latest_did)
+            try:
+                r = self.session.get(self.url + record.latest_did)
```

Comment:
Do you need a `resp.raise_for_status()` here? Also do you need a `check_response` call after the try/except?